src/app.scss

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.masonry-container {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 20px;
+  margin-top: 80px;
+  columns: 6;
+  column-gap: 14px;
+
+  &__flag {
+    height: 10px;
+  }
+
+  @media screen and (max-width: 576px) {
+    columns: 2;
+  }
+
+  @media screen and (min-width: 576px) and (max-width: 768px) {
+    columns: 3;
+  }
+
+  @media screen and (min-width: 768px) and (max-width: 992px) {
+    columns: 4;
+  }
```

Comment:
Hi @linrz John at Wiredcraft here. Thanks for your PR :)

Just a few questions:
* How serious are you about your stylistic choice of `semi: false` in your prettier config 😅 
* The use of css columns does successfully implement the masonry effect visually, but, as you mentioned in your readme, it functionally negates the lazy loading behavior. How would you implement the "real" lazy loading behavior without a lib?

Reply:
1. I think JavaScript supports no semicolon, there is no need to add a semicolon deliberately. ​​There are many languages ​​with or without a semicolon, such as Go, Scala, Ruby, Python, Swift and so on. Just a personal habit, for example, there is no semicolon in the `vue.js` soucre code, because of `Evan You` dislike adding semicolons. This problem is similar to the use of trailing commas in `Dart`. Although it is not beautiful, it is optional.

2. In order to complete the lazy loading in the task, I will fix the order of the pictures in each column, instead of automatically arranging them by the browser’s column layout. Because the height of the newly added pictures is inconsistent, the layout order of the columns that have been arranged before will be disordered. For example, the picture newly added that cannot fit at the end of the first column will be moved to the second column by browser's column behavior.

	Without a lib, if the browser does not support `IntersectionObserver`, the current window scroll area position and the top of the picture element in the HTML layout will be detected when scrolling(throttle), and if the top of the picture placeholder element appears in the window, the src of the picture will be assigned to load it.